docs/Makefile

```diff
@@ -6,7 +6,7 @@ ifneq ($(EXAMPLES_PATTERN),)
 endif
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W -j auto $(EXAMPLES_PATTERN_OPTS)
+SPHINXOPTS    = -j auto $(EXAMPLES_PATTERN_OPTS)
```

Comment:
Hi @oke-aditya , we should not remove the `-W` flag. What warnings were you getting?

Reply:
Oh no I commited wrong files.